puppetboard/utils.py

```diff
@@ -94,7 +94,10 @@ def prettyprint(value):
     for e in value:
         html += "<tr>"
         for k in e:
-            html += "<td>" + formatvalue(e[k]) + "</td>"
+            if k == "certname":
+                html += "<td> <a href = '/node/" + formatvalue(e[k]) + "' >" + formatvalue(e[k]) + "</a></td>"
```

Comment:
Please use `url_for()` Flask's function to generate the target URL. See https://flask.palletsprojects.com/en/1.1.x/quickstart/#url-building for more info on how and why. :)

Reply:
Thanks for the pointer, I think the code style is also making the linter happy, but I'm really a newbie regarding unit testing, can you please point me in the right direction there?